src/libstat/backends/redis_backend.c

```diff
@@ -1325,13 +1325,10 @@ rspamd_redis_connected (redisAsyncContext *c, gpointer r, gpointer priv)
 				}
 			}
 			else {
-				if (!rt->err) {
```

Comment:
Why is it needed? `runtime` is different for each classifier, so what is the purpose of this change?

Reply:
This is to avoid such case: any classifier is skipped, it throws out an error, then the result of the learn is errored.  

Ideally, it should be reporting the result for each classifier.

And "skipped" may not be an error, as it's an expected action defined by config.

Reply:
Ok, I see, I do agree with this point.